src/main/java/com/redhat/cloud/notifications/db/EndpointResources.java

```diff
@@ -347,21 +346,39 @@
                 });
     }
 
+    public Uni<Void> loadProperties(List<Endpoint> endpoints) {
+        // Group endpoints in types and load in batches for each type.
+        Set<Endpoint> endpointSet = new HashSet<>(endpoints);
+
+        return Uni.combine().all().unis(
+                this.loadTypedProperties(WebhookProperties.class, endpointSet, EndpointType.WEBHOOK)
+        ).discardItems();
```

Comment:
This will become a problem when the `EndpointType.CAMEL` is added because `Uni.combine()` may execute SQL queries concurrently with the same `Mutiny.Session`.

Chaining the `loadTypedProperties` calls will produce the expected result without any concurrency issue:

```
return loadTypedProperties(WebhookProperties.class, endpointSet, EndpointType.WEBHOOK)
        .chain(() -> loadTypedProperties(WebhookProperties.class, endpointSet, EndpointType.CAMEL));
```